index.html

```diff
@@ -4,9 +4,234 @@
+			</button>
+			<nav class="header-nav-mobile hidden">
+				<button type="button" class="btn-close-nav">
+					<img src="./img/icon/close-icon.svg" alt="cancel icon">
+				</button>
+				<ul>
+					<li>
+							<a href="./index.html" >
+								Home
+							</a>
+						</li>
+					<li>
+						<a href="./about.html" >
+							About
+						</a>
+					</li>
+					<li>
+						<a href="#" >
+							Program
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							Join
+						</a>
+					</li>
+					<li>
+						<a href="#" >
+							Sponsor
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							News
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							Ticket
+						</a>
+					</li>
+				</ul>
+			</nav>
+		</div>
+		<div class="headline">
+			<h3>
+				"The biggest International health care trade show in the east African region"
+			</h3>
+			<h1>
+				ETHIO HEALTH <br>EXHIBITION AND CONGRESS 2023
+			</h1>
+			<p class="headline-paragraph-description">
+				Ethio Health is the largest gathering of healthcare and trade professionals in the country. The 2023 edition of the
+				event is expected to welcome nearly 100 exhibiting companies and 4,000 + attendees.
+			</p>
+			<h2 class="headline-date">
+					2023.02.23(THU) ~ 25(SAT)
+			</h2>
+			<p class="headline-paragraph-location">
+					@ Ethiopian Skylight Hotel
+			</p>
+		</div>
+	</div>
+</header>
+<main>
+	<div class="container">
+		<div class="main-title">
+			<p>Main Program</p>
+		</div>
+		<div class="main-program-container">
+			<div class="main-cards">
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="./img/icon/lecture-icon.png">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Lecture
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Listen to the speakers from various countries about the messeges of sharing and opening
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/exhibit-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Exhibition
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Appreciate various creations applying CC license of artists, organized from Art Center
+							Nabi.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/forum-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Forum
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Have the time to share your thoughts and opinions with experts for each topic.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/workshop-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Workshop
+						</p>
```

Comment:
- [ ] Please make sure to fix the card title and text in the mobile version they should match the design, you can add some space between the card title and the text or you can use a responsive font size in `vw`. 
![image](https://user-images.githubusercontent.com/74744787/218276606-113c87c4-cded-41c6-9678-3297dc5a32f1.png)